unlock-protocol-com/blog/gitcoin-round-15.md

```diff
@@ -0,0 +1,108 @@
+our ecosystem going from 14 to 28.
+
+![Gitcoin's GR 15](/images/blog/gitcoin-round-15.png)
+
+Not only were we excited to see the number of grant applications increase in our
+ecosystem we were really excited to see the diversity in the types of initiatives 
+tagging us in. 
+
+## Climate Change
+We're sure that launching Unlock Protocol on the [Celo](https://celo.org/) 
+network is why we saw many more projects this round focused on climate change. 
+[Bloom Network](https://bloomnetwork.org/) continues to be the most well 
+supported project in both climate change and in the entire ecosystem. So far 
+they've raised more than $81k through Gitcoin Grants and we're so glad to see the 
+progress they've made with that funding. Some other projects with great potential 
+for climate impact are [Green Academy](https://green-academy.webflow.io/), 
+[Plants & Pillas](https://www.plantsandpillars3.com/), 
+[dMeter](https://dmeter.super.site/) and [Foresta](https://foresta.live/).
+
+## DeSci
+
+Some impressive projects leveraging Unlock Protocol for decentrailized science, 
+or DeSci, tagged us in this round. 
+[Omni Analytics](https://twitter.com/omnianalytics) maintaining the top spot for 
+community support in this category, has continued to use the funding they've 
+recieved by adding more content to their "Learning Data Science on the Ethereum 
+Blockchain" course. Congratulations to 
+[Active Inference Institute](https://www.activeinference.org/) who caught 
+community attention with their plans to create a first of it's kind open source 
+package Active Blockference which aims to implement Active Inference simulations 
+for congnitive modeling for the blockchain space, use Active Blockreference to 
+understand behavior in ETH/Web3 ecosystems and improve educational materials 
+around use of Active Blockreference. 
+
+## web3 Social
+
+We saw an increase in the number of web3 social projects taking advantage of 
+the power of Unlock Protocol too! [Nimi](https://nimi.io/) was a crowd favorite 
+with an dApp that allows you to aggregate your soical profiles. Other notable 
+projects include [Cobogo](https://cobogo.io) for creator profiles and 
+fundraising for the creator economy, [Maven Foundation](https://maven.foundation/) 
+is continuing their work on building tools to empower social influencers and 
+[MeTag](https://www.getmetag.io) is creating a scannable QR tag which is also NFC 
+enabled to help users share their web3 wallet information and social profiles. 
+[Headline](https://viaheadline.xyz/) a decentrailized publishing platform which 
+got it's start with a [UDT Grant](https://docs.unlock-protocol.com/governance/grants-bounties-and-matchings) from Unlock Protocol jumped onto the Gitcoin 
+platform this round to continue their work.
+
+## Education & Security
+
+[VestLab](VestLab.io) is aiming to get you all the information about a crypto 
+project before you invest. While [White Hat DAO](https://whitehatdao.com) is 
```

Comment:
don't need "While..." at the beginning here; please remove (ok to just start with White Hat DAO)